tests/test_action_ack_incident.py

```diff
@@ -0,0 +1,11 @@
+from ack_incident import AckIncident
+from st2tests.base import BaseActionTestCase
+
+
+class PagerDuytyAckIncidentActionTestCase(BaseActionTestCase):
```

Comment:
Nitpick: `Duyty` - @jjm Not sure if you want to correct this (and the rest of the testcase names). It's only in the name of the testcases, so it doesn't affect anything user-facing.

Reply:
Boo! That’s annoying.  I’ll correct that later on tonight.